test/converter.js

```diff
@@ -739,3 +739,149 @@ test('definitions refs', async function (test) {
   const context = newContext();
   await testConversion(test, type, 'Ref', expectedType, context);
 });
+
+test('oneOf', async function (test) {
+  const type = {
+    id: 'OneOf',
+    type: 'object',
+    properties: {
+      anAttribute: { type: 'string' },
+      attribute: {
+        oneOf: [
+          {
+            type: 'object',
+            properties: {
+              first: { type: 'string' }
+            }
+          },
+          {
+            type: 'object',
+            properties: {
+              second: { type: 'string' }
+            }
+          }
+        ]
+      }
+    }
+  };
+
+  const expectedType = `
+    type OneOfAttributeSwitch0 {
+      first: String
+    }
+    type OneOfAttributeSwitch1 {
+      second: String
+    }
+    union OneOfAttribute = OneOfAttributeSwitch0 | OneOfAttributeSwitch1
+    type OneOf {
+      anAttribute: String
+      attribute: OneOfAttribute
+    }
+    input OneOfIn {
+      anAttribute: String
+    }
+  `;
+
+  const context = newContext();
+  await testConversion(test, type, 'OneOf', expectedType, context);
+});
+
+test('oneOf', async function (test) {
```

Comment:
What is the difference between this test and the one above?

Reply:
Looks like a rebase issue^^ I'll amend